packages/ui/src/pages/Home/MultisigActionMenu.tsx

```diff
@@ -8,9 +8,13 @@ import { useModals } from '../../contexts/ModalsContext'
 
 interface MultisigActionMenuProps {
   withSendButton?: boolean
+  withSettingsButton?: boolean
```

Comment:
This props is never used afaics, and doesn't have the right name anyways, these aren't settings. Should we remove this props altogether?